src/_store/BootProvider.js

```diff
@@ -1,5 +1,6 @@
 import React, { Children, PureComponent, PropTypes } from 'react';
 import { connect } from 'react-redux';
+import Intl from 'intl';
```

Comment:
How large is it?
It might not be worth statically increasing our bundle size just for iPod.
Also this can be reported as bug to polyfill.io and they will likely fix it.